core/lib/types/Smash.agda

```diff
@@ -6,13 +6,11 @@ open import lib.types.Coproduct
 open import lib.types.Paths
 open import lib.types.Span
 open import lib.types.Pushout
-open import lib.types.Cofiber
 open import lib.types.Sigma
-open import lib.types.Wedge
 
-module lib.types.Smash {i j} where
+module lib.types.Smash where
```

Comment:
@timjb Is this due to some type-checking issue or just coding style (which is fine)? It seems all the following still has "{i j}" as the prefix.

Reply:
That's a leftover of some development that I did in this file (which required generalizing the universe levels) which I later moved to a separate (still uncommitted) file. You're right that this change is not needed (anymore). Still, I like the way it is now better: Introducing the universe levels together with the types they're for seems cleaner to me.

Reply:
I actually agree! I was just curious if there's any other reason. :smile: